flush entity a publish do Rabbita v onShutdown[]

michal-loksik opened the pull request.

Commit:
flush entity a publish do Rabbita v onShutdown[]

michallohnisky commented on src/Service.php:
```diff
@@ -13,13 +13,23 @@ class Service extends \Nette\Object {
 		$this->bunny = $bunny;
+
+		$application->onShutdown[] = function () {
+			foreach ($this->callbacks as $callback) {
```
Přejmenuj `callbacks` na `onShutdown` a taky zruš foreach a zavolej `$this->onShutdown();`

michal-loksik replied:
@michallohnisky 
upravene

Commit:
- premenovane pole $callbacks na $onShutdown
- zruseny foreach .. volane $this->onShutdown()

Commit:
property $onShutdown zmenena na public

michal-loksik commented:
@michallohnisky 

- property $onShutdown zmenena na public
- zruseny foreach a pouzite $this->onShutdown()
